Testing/Exec/ReactEval_C/GPU_misc.H

```diff
@@ -56,11 +56,7 @@ initialize_data(int i, int j, int k, int fuel_id,
 	    frcExt(i,j,k,n) = 0.0;
 	}
 	rhoY(i,j,k,NUM_SPECIES) = temp; 
-#if defined(USE_SUNDIALS_PP)
 	rhoE(i,j,k) = energy * density; 
```

Comment:
This one seems very strange.  Is is really the case that the actual evolved variable is (was) different based non whether CUDA SUNDIALS is used?  And if so, is it correct that that is no longer the case??